plugin/src/main/java/org/opensearch/ml/model/MLModelManager.java

```diff
@@ -1079,6 +1042,65 @@ public void deployModel(
         }
     }
 
+    private void deployRemoteOrBuiltInModel(MLModel mlModel, Integer eligibleNodeCount, ActionListener<String> wrappedListener) {
+        String modelId = mlModel.getModelId();
+        setupModelRateLimiter(modelId, eligibleNodeCount, mlModel.getModelRateLimiterConfig());
+        if (mlModel.getConnector() != null || FunctionName.REMOTE != mlModel.getAlgorithm()) {
```

Comment:
What does this mean?

Reply:
As method name said, it only covered remote and built-in model like Kmeans. Local model is covered in other part